### PR TITLE
TEMP-1 - Various fixes

### PR DESCRIPTION
**Pull request by troyfontaine**
# OVERVIEW

Yep, not a jira tag.

This includes a bunch of fixes/tweaks/documentation improvements.
- Breaks sub_domain/hostname out from domain
- Tweaks to Caddyfile configuration to allow for the proxy to nginx to fail without breaking Caddy
- Output that shows the full URL to login to the code server installation
- Tweaks to the scenario explanation for nginx
- Adds a success criteria to the scenario description for nginx

## WHY THE PULL REQUEST

Various fixes to the documentation and configuration.

## HOW TO QA

Please provide some steps on how to QA this fix

1. Step 1
2. Step 2
3. Step 3
4. Step 4

## Checklist

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +74,12 @@ The testing performed using these tools is intended to be used in conjunction wi
 
 Testing technical staff can monitor the instance by connecting via SSH to the instance and to provide additional support to the candidate.
 
+## Known Issues
+
+Due to the throwaway nature of this task and repository, the terraform state is handled locally in addition, the private key for the instance and the password for the code-server are treated insecurely.
```
Comment (GrooveStomp) [inline README.md]: > handled locally; in addition,

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The included AMIs built with this default repository consist of:
+
+- Broken Nginx
+- Broken Apache
+- Broken Python Web App
+- Broken Ruby Web App
+- Broken Bash Script
+
+### Broken Apache
```
Comment (GrooveStomp) [inline docs/README.md:13]: Missing summary for Broken Nginx too?

File: README.md (hunk 1 above)
Comment (troyfontaine) [inline README.md]:
```suggestion
Due to the throwaway nature of this task and repository, the terraform state is handled locally, in addition, the private key for the instance and the password for the code-server are treated insecurely.
```

File: docs/README.md (hunk 2 above)
Comment (troyfontaine) [reply docs/README.md:13]: I didn't want to commit this file, oops.

**Review by GrooveStomp — APPROVED** [review]

**Pull request merged**